recipes/opentelemac/meta.yaml

```diff
@@ -40,6 +40,14 @@ requirements:
     - metis
     - numpy >=1.25,<1.26
   run:
+    - {{ compiler('fortran') }}
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - openmpi
```

Comment:
we could also keep 
```
        - {{ mpi }} 
        - {{ mpi }}-mpifort 
```
to have more compiling possibilities

Reply:
Yeah but let's do this once it's merged in feedstock